google/ggrc-core
# GGRC-6219 Assessment count doesn't match in audit summary

# Issue description
Audit Summary statistic doesn't match Assessements tab in "In Review" state.

I found that main reason is "verified_date" population for Assessments in "In Review" state. "verified_date" should be populated only for Assessments in "Verified" and "Completed" states, but some assessments in "In Review" states also had "verified_date". Main reason was incorrect population "verified_date" by import assessments

I added migration to revert "verified_date" to NULL for Assessments "In Review" state
I fixed "verified_date" population regarding assessment "state"

More details in the ticket

# Steps to test the changes

Create Assessment and change state ("Not Started", "In Progress" and etc.).
Import Assessments with various states.
Check Audit Summary page for Assessments with various states, statistic should be correct
"verified_date" should be set only for assessment in "Verified" and "Completed" state. For another states "verified_date" should be NULL

# Solution description
Add migration to revert "verified_date" to NULL for Assessments "In Review" state
Fix "verified_date" population regarding assessment "state"

# Sanity checklist


# PR Review checklist
